### PR TITLE
Connor/bug-fixing

### PR DESCRIPTION
**Pull request by connorreinhold**
Fix a bug where account info would not save when not viewing the fragment upon completion

Fixed display of middle tab times

Fixed dining hall ids to correctly display dining halls

### CODE REVIEW

**Comment by connorreinhold** [comment]
The old API key is in there which is fine. We have created a new one

File: app/src/main/java/com/cornellappdev/android/eatery/ExpandableListAdapter.java [hunk 1]
```diff
@@ -106,15 +108,31 @@ public View getGroupView(int diningHallNumber, boolean isExpanded, View view,
...
             openText.setPadding(0, 0, 8, 0);
-            openText.setText(R.string.closed);
-            openText.setTextColor(ContextCompat.getColor(mContext, R.color.red));
+            EateryBaseModel.Status currentStatus = dm.getCurrentStatus();
             if (interval.containsTime(currentTime)) {
-                openText.setText(R.string.open);
-                openText.setTextColor(ContextCompat.getColor(mContext, R.color.green));
+                openText.setText(currentStatus.toString());
+                if (currentStatus == EateryBaseModel.Status.OPEN) {
+                    openText.setText(currentStatus.toString());
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/ExpandableListAdapter.java]: remove this line or line at 116

File: app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import android.text.SpannableString;
 import android.text.style.RelativeSizeSpan;
 import android.util.DisplayMetrics;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java]: please remove

File: app/src/main/java/com/cornellappdev/android/eatery/ExpandableListAdapter.java [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import android.content.Context;
 import android.graphics.Typeface;
 import android.text.SpannableString;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/ExpandableListAdapter.java]: please remove, and any other imports that are not being used

File: app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package com.cornellappdev.android.eatery.model;
 
 import android.content.Context;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java]: please remove

File: app/src/main/java/com/cornellappdev/android/eatery/network/QueryUtilities.java [hunk 5]
```diff
@@ -19,7 +19,7 @@
 public final class QueryUtilities {
 
     private final static HashSet<Integer> DINING_HALL_IDS =
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/network/QueryUtilities.java]: we should leave a comment about where are you getting these ids from

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -124,7 +124,7 @@
     <string name="history">History</string>
 
     <!-- Below values need to be updated before each launch -->
-    <string name="encryption_key">hi</string>
-    <string name="encryption_salt">you</string>
-    <string name="google_maps_key">guys!</string>
+    <string name="encryption_key">a</string>
```
Comment (ningning621) [inline app/src/main/res/values/strings.xml]: are we deprecating this for a `keys.properties` file?

Comment (ningning621) [reply app/src/main/res/values/strings.xml]: if not, let's create a git issue

Comment (connorreinhold) [reply app/src/main/res/values/strings.xml]: ye ill create an issue

Comment (ningning621) [reply app/src/main/res/values/strings.xml]: #198

**Pull request merged**